documentation/publish/signing-and-distribution-overview.md

```diff
@@ -68,7 +68,7 @@ Unsigned extensions can be installed in [Developer Edition](https://www.mozilla.
 
 Mozilla signs add-ons through the [AMO](https://addons.mozilla.org) website and provides three methods for submitting your add-on for signing:
 
-- upload your add-on through the [Developer Hub on AMO](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on).
+- upload your add-on through the [Developer Hub on AMO](https://developer.mozilla.org/docs/Mozilla/Add-ons/Distribution/Submitting_an_add-on).
```

Comment:
Please change link to `/documentation/publish/submitting-an-add-on/`.

Reply:
Same here

Reply:
Good question -- we can be more specific and link to `/documentation/publish/submitting-an-add-on/#listing-on-amo`.